Cargo.toml

```diff
@@ -102,3 +102,6 @@ harness = false
 [[bench]]
 name = "pedersen_hash"
 harness = false
+
+[patch.crates-io]
+zcash_note_encryption = { version = "0.4", git = "https://github.com/QED-it/zcash_note_encryption", branch = "resolve_zcash_pr2_issues"  }
```

Comment:
Are we going to merge zcash_note_encryption first and use zsa1 here?

Reply:
yes